detection-rules/link_login_or_captcha.yml

```
@@ -6,15 +6,14 @@ severity: "high"
 source: |
   type.inbound 
   and any(body.links, 
-      all([beta.linkanalysis(.)], (
-          .credphish.contains_login
-          or .credphish.contains_captcha
-      )
-      and (
-          .effective_url.domain.root_domain in $free_subdomain_hosts
-          or ..href_url.domain.root_domain in $free_subdomain_hosts
-      ))
+      beta.linkanalysis(.).credphish.contains_login
+      or beta.linkanalysis(.).credphish.contains_captcha
+  )
+  and any(body.links,
+          beta.linkanalysis(.).effective_url.domain.root_domain in $free_subdomain_hosts
+          or .href_url.domain.root_domain in $free_subdomain_hosts
```

got changed on these too, idk if my suggestions didn't work or something
```suggestion
      beta.linkanalysis(.).effective_url.domain.root_domain in $free_subdomain_hosts
      or .href_url.domain.root_domain in $free_subdomain_hosts
```